ghost/core/core/server/services/mentions/BookshelfMentionRepository.js

```diff
@@ -107,7 +108,8 @@ module.exports = class BookshelfMentionRepository {
             resource_id: mention.resourceId?.toHexString(),
             resource_type: mention.resourceId ? 'post' : null,
             payload: mention.payload ? JSON.stringify(mention.payload) : null,
-            deleted: Mention.isDeleted(mention)
+            deleted: Mention.isDeleted(mention),
+            verified: mention.verified || false
```

Comment:
I don't think we need this check - the Mention entity always has a `boolean` for the verified property